automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookTestUtils.scala

```diff
@@ -332,11 +351,25 @@ trait NotebookTestUtils extends LeonardoTestUtils {
     }
   }
 
-  def getObject(workspaceBucketName: GcsBucketName, notebookName: GcsBlobName): IO[Option[String]] = {
+  def getObjectAsString(workspaceBucketName: GcsBucketName, notebookName: GcsBlobName): IO[Option[String]] = {
     google2StorageResource.use {
       google2StorageDAO =>
         google2StorageDAO.unsafeGetObject(workspaceBucketName, notebookName, None)
     }
   }
 
+  def getObjectAsFile(workspaceBucketName: GcsBucketName, notebookName: GcsBlobName): File = {
```

Comment:
google2 has a https://github.com/broadinstitute/workbench-libs/blob/develop/google2/src/main/scala/org/broadinstitute/dsde/workbench/google2/GoogleStorageInterpreter.scala#L80 that allows you to down load an object to a local file